.github/workflows/tests.yml

```diff
@@ -59,19 +59,26 @@ jobs:
         if: matrix.conda-version == 'canary'
         run: |
           conda install -y -c conda-canary/label/dev conda
-
       - name: Install default Conda if release version
         if: matrix.conda-version == 'release'
         run: |
           conda install -y conda
-
       - name: Source Scripts
         run: |
           source ci/github/install_conda_build_test_deps
           pip install -e .
+          pip install setuptools wheel twine
+          pip install pytest-html
+          pip install allure-pytest
           conda info -a
-          mkdir ./pytest-replay
 
+          echo "Creating Replay and Result Report Folders For Test Summaries..."
+          mkdir -p ./allure-results
```

Comment:
we will only need this one directory allure-results, if this job is only collecting test results. Other folders are only necessary when generating the pretty report.

Reply:
yes that is correct.. for that should be done for 3 Operating System